.drone-multiarch.yml

```diff
@@ -0,0 +1,106 @@
+kind: pipeline
+name: amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: build
+    image: rancher/dapper:1.11.2
```

Comment:
switch to dapper:v0.4.1 which is multiarch now

Reply:
Updated to dapper:v0.4.1